src/use-swr.ts

```diff
@@ -196,20 +74,23 @@ export const useSWRHandler = <Data = any, Error = any>(
 
   // Get the current state that SWR should return.
   const cachedData = cache.get(key)
-  const data = isUndefined(cachedData) ? initialData : cachedData
+  const fallbackData = isUndefined(initialData)
+    ? fallbackValues[key]
+    : initialData
```

Comment:
Are there plans to remove `initialData` entirely in v1? This seems like a good opportunity, replace it with `fallback`/`fallbackValues`

Reply:
Yeah its use case is already covered completely by `fallbackValues`, but I'm thinking about deprecating it with `fallbackValue` for better fit:

```js
// we can put multiple values here, but it's longer
useSWR(key, fetcher, { fallbackValues: { [key]: 'value' } }) 

// same as initialData, but the name is clearer
useSWR(key, fetcher, { fallbackValue: 'value' })
```

Maybe we should deprecating `initialData` in v1, otherwise we'll have to wait for v2.